src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const valuesList = sourceString
+    .split(';')
+    .filter((el) => el.trim())
+    .map((e) => e.split(':').map((s) => s.trim()))
+    .reduce((acc, [key, value]) => {
+      acc[key] = value;
+
+      return acc;
+    }, {});
+
+  return valuesList;
```

Comment:
The variable name `valuesList` is not descriptive of the content it holds. According to the [CODE STYLE] checklist, avoid generic names like `array`, `object`, `result`. A more descriptive name would be `styleObject` or `cssProperties` to reflect that it represents the styles converted to an object.